apps/web/src/screens/Dashboard/index.tsx

```diff
@@ -28,6 +29,12 @@ export const DashboardScreen: FC = () => {
 		await router.navigate('/');
 	};
 
+	useEffect(() => {
+		if (pathname !== setting?.latestLocation) {
+			setPathname(pathname);
+		}
+	}, [pathname]);
+
```

Comment:
we already agreed to move `setPathname` outside `DashboardScreen` in the last discussion. However, is that solution possible?

Reply:
> we already agreed to move `setPathname` outside `DashboardScreen` in the last discussion. However, is that solution possible?

We discussed about it and all of us agree to put it in `DashboardScreen`